docker-compose.yaml

```
@@ -9,12 +9,39 @@ services:
       dockerfile: Dockerfile
     ports:
       - "8080:3000"
+    depends_on:
+      - robin-db
+    environment:
+      DB_HOST: db
+      DB_PORT: 3306
+      DB_USER: root
+      DB_PASSWORD: password
+      DB_NAME: robin-development
     volumes:
       - ./configs:/root/configs
     restart: unless-stopped
     networks:
       - rha_local
 
+  robin-db:
+    container_name: robin-db
+    image: mysql:8.4.0
+    cap_add:
+      - SYS_NICE
+    restart: always
+    environment: 
+      - MYSQL_DATABASE=robin-development
```

```suggestion
      - MYSQL_DATABASE=rha
```

suggest using a generic name that is consistent across environments

---

Got it.